content-src/components/NewTabPage/NewTabPage.js

```diff
@@ -56,14 +56,15 @@ const NewTabPage = React.createClass({
           title="Welcome to new tab"
           body="Firefox will use this space to show your most relevant bookmarks, articles, videos, and pages you've recently visited, so you can get back to them easily."
           label="Identifying your Highlights" />
-        <div className={classNames("show-on-init", {on: this.props.isReady})}>
+        <div className={classNames("show-on-init", "on")}>
           <section>
-            <TopSites page={PAGE_NAME} sites={props.TopSites.rows} showHint={props.TopSites.showHint} />
+            <TopSites placeholder={!this.props.isReady} page={PAGE_NAME}
+              sites={props.TopSites.rows} showHint={props.TopSites.showHint} />
           </section>
 
           <section>
-            <Spotlight page={PAGE_NAME} length={HIGHLIGHTS_LENGTH}
-              sites={props.Highlights.rows} />
+            <Spotlight placeholder={!this.props.isReady} page={PAGE_NAME}
+              length={HIGHLIGHTS_LENGTH} sites={props.Highlights.rows} />
```

Comment:
Yes; that's the design.  See scenario 2A in https://mozilla.invisionapp.com/share/D995GXJVC#/201431382_First_Run_Messaging Scenario